src/runtime/4.8/windowsservercore-2004/Dockerfile

```diff
@@ -8,17 +8,7 @@ ENV `
     COMPLUS_NGenProtectedProcess_FeatureEnabled=0
 
 RUN `
-    # Apply latest patch
-    curl -fSLo patch.msu http://download.windowsupdate.com/d/msdownload/update/software/secu/2021/01/windows10.0-kb4601050-x64-ndp48_04c8736f250c06b3787d7d9ec366f011c6f8ca91.msu `
-    && mkdir patch `
-    && expand patch.msu patch -F:* `
-    && del /F /Q patch.msu `
-    && dism /Online /Quiet /Add-Package /PackagePath:C:\patch\windows10.0-kb4601050-x64-ndp48.cab `
-    && rmdir /S /Q patch `
-    `
     # Ngen top of assembly graph to optimize a set of frequently used assemblies
-    && %windir%\Microsoft.NET\Framework64\v4.0.30319\ngen install "Microsoft.PowerShell.Utility.Activities, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" `
+    %windir%\Microsoft.NET\Framework64\v4.0.30319\ngen install "Microsoft.PowerShell.Utility.Activities, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"
     # To optimize 32-bit assemblies, uncomment the next line
     # && %windir%\Microsoft.NET\Framework\v4.0.30319\ngen install "Microsoft.PowerShell.Utility.Activities, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" `
-    && %windir%\Microsoft.NET\Framework64\v4.0.30319\ngen update `
```

Comment:
This looks good.  Given the number of issues in the past with the Windows base images, do you think it would be worthwhile to have a unit test that validates the ngen queue is empty?  If so I don't think it is necessary for this PR and could be tracked separately.

Reply:
We already do:

https://github.com/microsoft/dotnet-framework-docker/blob/d4095f39459c58f1951abbf21cf0a2b95458fac4/tests/Microsoft.DotNet.Framework.Docker.Tests/RuntimeOnlyImageTests.cs#L52